mockwebserver/src/main/java/okhttp3/mockwebserver/MockWebServer.java

```diff
@@ -106,7 +106,7 @@
     MwsDuplexAccess.instance = new MwsDuplexAccess() {
       @Override public void setBody(
           MockResponse mockResponse, DuplexResponseBody duplexResponseBody) {
-        mockResponse.setBody(duplexResponseBody);
+        mockResponse.setBody$mockwebserver(duplexResponseBody);
```

Comment:
I'm not sure how safe this is to rely on...

Reply:
It does seem to be unstable, `./gradlew clean build `can always pass, but IDEA occasionally can't find references. I removed all the `internal` in this class and fixed the reference.